cordova-js-src/exec.js

```diff
@@ -41,6 +41,20 @@ var execProxy = require('cordova/exec/proxy');
  */
 module.exports = function (success, fail, service, action, args) {
 
+    // Handle the case when we have an old version of splashscreen plugin to avoid the API calls failures
+    if (service === 'SplashScreen') {
+        var pluginsVersions = require("cordova/plugin_list").metadata;
+        var splashscreenVersion = pluginsVersions['cordova-plugin-splashscreen'];
+        var MIN_SPLASHSCREEN_SUPPORTED_VER = 4;
+        if (splashscreenVersion && parseInt(splashscreenVersion[0], 10) < MIN_SPLASHSCREEN_SUPPORTED_VER) {
```

Comment:
This regards to the plugin version and not to the platform' one,  i.e. it should work starting with cordova-plugin-splashscreen@4 (as it will just redirect API calls to the platform).